### PR TITLE
update picture of diagnosis doc

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-sql-tuning-overview.md [hunk 1]
```diff
@@ -30,10 +30,14 @@ Note that on this sub-tab, SQL queries with the same structure (even if the quer
...
+
+![Details0](/media/dashboard/dashboard-statement-detail0.png)
+
 - Execution plan list: if a SQL statement has more than one execution plan, the list is displayed. You can select different execution plans and the details of the selected execution plan are displayed at the bottom of the list. If there is only one execution plan, the list will not be displayed.
-- Execution plan details: shows the details of the selected execution plan. It collects the execution plans of such SQL type and the corresponding execution time from several perspectives to help you get more information. See [Execution plan in details](https://docs.pingcap.com/tidb/stable/dashboard-statement-details#statement-execution-details-of-tidb-dashboard) (area 3 in the image below).
+- Execution plan details: shows the details of the selected execution plan. It collects the execution plans of such SQL type and the corresponding execution time from several perspectives to help you get more information. See [Execution plan in details](https://docs.pingcap.com/tidb/stable/dashboard-statement-details#statement-execution-details-of-tidb-dashboard) .
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-sql-tuning-overview.md:37]:
```suggestion
- Execution plan details: shows the details of the selected execution plan. It collects the execution plans of such SQL type and the corresponding execution time from several perspectives to help you get more information. See [Execution plan in details](https://docs.pingcap.com/tidb/stable/dashboard-statement-details#statement-execution-details-of-tidb-dashboard).
```

File: tidb-cloud/tidb-cloud-sql-tuning-overview.md [hunk 2]
```diff
@@ -30,10 +30,14 @@ Note that on this sub-tab, SQL queries with the same structure (even if the quer
 You can view some key information in **Statement**.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-sql-tuning-overview.md:30]:
```suggestion
You can view some key information in **SQL Statement**.
```

File: tidb-cloud/tidb-cloud-sql-tuning-overview.md [hunk 3]
```diff
@@ -30,10 +30,14 @@ Note that on this sub-tab, SQL queries with the same structure (even if the quer
...
 
 - SQL statement overview: including SQL digest, SQL template ID, the time range currently viewed, the number of execution plans, and the database where the execution takes place.
+
+![Details0](/media/dashboard/dashboard-statement-detail0.png)
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-sql-tuning-overview.md:34]:
```suggestion
    ![Details0](/media/dashboard/dashboard-statement-detail0.png)
```

File: tidb-cloud/tidb-cloud-sql-tuning-overview.md [hunk 4]
```diff
@@ -30,10 +30,14 @@ Note that on this sub-tab, SQL queries with the same structure (even if the quer
...
-- Execution plan details: shows the details of the selected execution plan. It collects the execution plans of such SQL type and the corresponding execution time from several perspectives to help you get more information. See [Execution plan in details](https://docs.pingcap.com/tidb/stable/dashboard-statement-details#statement-execution-details-of-tidb-dashboard) (area 3 in the image below).
+- Execution plan details: shows the details of the selected execution plan. It collects the execution plans of each SQL type and the corresponding execution time from several perspectives to help you get more information. See [Execution plan in details](https://docs.pingcap.com/tidb/stable/dashboard-statement-details#statement-execution-details-of-tidb-dashboard).
 
-![Details](/media/dashboard/dashboard-statement-detail.png)
+![Details1](/media/dashboard/dashboard-statement-detail1.png)
+![Details2](/media/dashboard/dashboard-statement-detail2.png)
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-sql-tuning-overview.md:40]:
```suggestion
    ![Details1](/media/dashboard/dashboard-statement-detail1.png)
    ![Details2](/media/dashboard/dashboard-statement-detail2.png)
```

File: tidb-cloud/tidb-cloud-sql-tuning-overview.md [hunk 5]
```diff
@@ -30,10 +30,14 @@ Note that on this sub-tab, SQL queries with the same structure (even if the quer
...
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-sql-tuning-overview.md:41]: To be added:
```suggestion

- Related Slow Query:

```

File: tidb-cloud/tidb-cloud-sql-tuning-overview.md [hunk 6]
```diff
@@ -30,10 +30,14 @@ Note that on this sub-tab, SQL queries with the same structure (even if the quer
 You can view some key information in **Statement**.
 
 - SQL statement overview: including SQL digest, SQL template ID, the time range currently viewed, the number of execution plans, and the database where the execution takes place.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-sql-tuning-overview.md:32]:
```suggestion
- **SQL Template**: including SQL digest, SQL template ID, the time range currently viewed, the number of execution plans, and the database where the execution takes place.
```

File: tidb-cloud/tidb-cloud-sql-tuning-overview.md [hunk 7]
```diff
@@ -30,10 +30,14 @@ Note that on this sub-tab, SQL queries with the same structure (even if the quer
...
+
+![Details0](/media/dashboard/dashboard-statement-detail0.png)
+
 - Execution plan list: if a SQL statement has more than one execution plan, the list is displayed. You can select different execution plans and the details of the selected execution plan are displayed at the bottom of the list. If there is only one execution plan, the list will not be displayed.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-sql-tuning-overview.md:36]: This description is different from the UI. Need updates.

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 8]
```diff
@@ -175,7 +175,7 @@ You can also use the AWS dashboard to configure the VPC peering connection.
 
     2. From the left navigation bar, open the **Peering Connections** page. On the **Create Peering Connection** tab, a peering connection is in the **Pending Acceptance** status.
 
-    3. Confirm the requester owner is TiDB Cloud (`380838443567` or `143458967504`). Right-click the peering connection and select **Accept Request** to accept the request in the **Accept VPC peering connection request** dialog.
+    3. Confirm the requester owner is TiDB Cloud (`380838443567` or `143458967504` or `730335266318`). Right-click the peering connection and select **Accept Request** to accept the request in the **Accept VPC peering connection request** dialog.
```
Comment (qiancai) [inline tidb-cloud/set-up-vpc-peering-connections.md:178]:
```suggestion
    3. Confirm the requester owner is TiDB Cloud (`380838443567`， `143458967504`, or `730335266318`). Right-click the peering connection and select **Accept Request** to accept the request in the **Accept VPC peering connection request** dialog.
```

File: tidb-cloud/tidb-cloud-sql-tuning-overview.md [hunk 9]
```diff
@@ -27,13 +27,19 @@ The TiDB Cloud console provides a **[SQL Statement](/tidb-cloud/tune-performance
...
-- Execution plan details: shows the details of the selected execution plan. It collects the execution plans of such SQL type and the corresponding execution time from several perspectives to help you get more information. See [Execution plan in details](https://docs.pingcap.com/tidb/stable/dashboard-statement-details#statement-execution-details-of-tidb-dashboard) (area 3 in the image below).
+- Execution plan details: shows the details of the selected execution plan. It collects the execution plans of each SQL type and the corresponding execution time from several perspectives to help you get more information. See [Execution plans](https://docs.pingcap.com/tidb/stable/dashboard-statement-details#execution-plans).
+
+    ![Details1](/media/dashboard/dashboard-statement-detail1.png)
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-sql-tuning-overview.md:39]: 这幅图是否属于 L36 的 Execution plan list

File: tidb-cloud/tidb-cloud-sql-tuning-overview.md [hunk 10]
```diff
@@ -27,13 +27,19 @@ The TiDB Cloud console provides a **[SQL Statement](/tidb-cloud/tune-performance
...
+    ![Details2](/media/dashboard/dashboard-statement-detail2.png)
 
-![Details](/media/dashboard/dashboard-statement-detail.png)
+- Related Slow Query:
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-sql-tuning-overview.md:42]: 这后面需要补充解释或图片吗

File: tidb-cloud/tidb-cloud-sql-tuning-overview.md (hunk 9 above)
Comment (qqqdan) [reply tidb-cloud/tidb-cloud-sql-tuning-overview.md:39]: 是的

File: tidb-cloud/tidb-cloud-sql-tuning-overview.md (hunk 10 above)
Comment (qqqdan) [reply tidb-cloud/tidb-cloud-sql-tuning-overview.md:42]: 后面这些是 SQL query，没有界面了。 Link 的文档中有详细的一些例子。

Comment (qiancai) [reply tidb-cloud/tidb-cloud-sql-tuning-overview.md:42]: 要不把冒号删除了，或者在冒号后加一两句话，现在一眼看过去，容易看成后面的内容写漏了😂
image

File: tidb-cloud/tidb-cloud-sql-tuning-overview.md (hunk 9 above)
Comment (qiancai) [inline tidb-cloud/tidb-cloud-sql-tuning-overview.md:39]:
```suggestion
```

File: tidb-cloud/tidb-cloud-sql-tuning-overview.md [hunk 11]
```diff
@@ -27,13 +27,19 @@ The TiDB Cloud console provides a **[SQL Statement](/tidb-cloud/tune-performance
...
 
-You can view some key information in **Statement**.
+You can view some key information in **SQL Statement**.
+
+- **SQL Template**: including SQL digest, SQL template ID, the time range currently viewed, the number of execution plans, and the database where the execution takes place.
+
+    ![Details0](/media/dashboard/dashboard-statement-detail0.png)
 
-- SQL statement overview: including SQL digest, SQL template ID, the time range currently viewed, the number of execution plans, and the database where the execution takes place.
 - Execution plan list: if a SQL statement has more than one execution plan, the list is displayed. You can select different execution plans and the details of the selected execution plan are displayed at the bottom of the list. If there is only one execution plan, the list will not be displayed.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-sql-tuning-overview.md:36]:
```suggestion
- Execution plan list: if a SQL statement has more than one execution plan, the list is displayed. You can select different execution plans and the details of the selected execution plan are displayed at the bottom of the list. If there is only one execution plan, the list will not be displayed.

    ![Details1](/media/dashboard/dashboard-statement-detail1.png)
```

File: tidb-cloud/tidb-cloud-sql-tuning-overview.md (hunk 9 above)
Comment (qiancai) [reply tidb-cloud/tidb-cloud-sql-tuning-overview.md:39]: 那这幅图可以挪到  L36 的 Execution plan list 后面

File: tidb-cloud/tidb-cloud-sql-tuning-overview.md [hunk 12]
```diff
@@ -27,13 +27,20 @@ The TiDB Cloud console provides a **[SQL Statement](/tidb-cloud/tune-performance
...
-- Execution plan details: shows the details of the selected execution plan. It collects the execution plans of such SQL type and the corresponding execution time from several perspectives to help you get more information. See [Execution plan in details](https://docs.pingcap.com/tidb/stable/dashboard-statement-details#statement-execution-details-of-tidb-dashboard) (area 3 in the image below).
 
-![Details](/media/dashboard/dashboard-statement-detail.png)
+    ![Details1](/media/dashboard/dashboard-statement-detail1.png)
+- Execution plan details: shows the details of the selected execution plan. It collects the execution plans of each SQL type and the corresponding execution time from several perspectives to help you get more information. See [Execution plans](https://docs.pingcap.com/tidb/stable/dashboard-statement-details#execution-plans).
+
+    ![Details2](/media/dashboard/dashboard-statement-detail2.png)
+
+- Related Slow Query:
```
Comment (qqqdan) [inline tidb-cloud/tidb-cloud-sql-tuning-overview.md:43]:
```suggestion
- Related Slow Query
```

File: tidb-cloud/tidb-cloud-sql-tuning-overview.md [hunk 13]
```diff
@@ -27,13 +27,21 @@ The TiDB Cloud console provides a **[SQL Statement](/tidb-cloud/tune-performance
...
-- Execution plan details: shows the details of the selected execution plan. It collects the execution plans of such SQL type and the corresponding execution time from several perspectives to help you get more information. See [Execution plan in details](https://docs.pingcap.com/tidb/stable/dashboard-statement-details#statement-execution-details-of-tidb-dashboard) (area 3 in the image below).
 
-![Details](/media/dashboard/dashboard-statement-detail.png)
+    ![Details1](/media/dashboard/dashboard-statement-detail1.png)
+
+- Execution plan details: shows the details of the selected execution plan. It collects the execution plans of each SQL type and the corresponding execution time from several perspectives to help you get more information. See [Execution plans](https://docs.pingcap.com/tidb/stable/dashboard-statement-details#execution-plans).
+
+    ![Details2](/media/dashboard/dashboard-statement-detail2.png)
+
+- Related Slow Query
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-sql-tuning-overview.md:44]:
```suggestion
- Related slow query
```

**Review by qiancai — APPROVED** [review]
LGTM

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 14]
```diff
@@ -175,7 +175,7 @@ You can also use the AWS dashboard to configure the VPC peering connection.
 
     2. From the left navigation bar, open the **Peering Connections** page. On the **Create Peering Connection** tab, a peering connection is in the **Pending Acceptance** status.
 
-    3. Confirm the requester owner is TiDB Cloud (`380838443567` or `143458967504`). Right-click the peering connection and select **Accept Request** to accept the request in the **Accept VPC peering connection request** dialog.
+    3. Confirm the requester owner is TiDB Cloud (`380838443567`， `143458967504`, or `730335266318`). Right-click the peering connection and select **Accept Request** to accept the request in the **Accept VPC peering connection request** dialog.
```
Comment (hfxsd) [inline tidb-cloud/set-up-vpc-peering-connections.md:178]:
```suggestion
    3. Confirm the requester owner is TiDB Cloud (`380838443567`，`143458967504`, or `730335266318`). Right-click the peering connection and select **Accept Request** to accept the request in the **Accept VPC peering connection request** dialog.
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**